man/systemd.network.xml

```diff
@@ -1770,6 +1770,15 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
 
         <!-- How to use the DHCP lease -->
 
+        <varlistentry>
+          <term><varname>Label=</varname></term>
+          <listitem>
+            <para>Specifies the label for the IPv4 address received from the DHCP server.
+            The label must consist only of 7-bit ASCII characteres and must be between 1 and 15 characteres long.
```

Comment:
characteres → characters.

maybe shorter: "The label must be a 7bit ASCII string with a length of 1…15 characters."

Reply:
@poettering fixed
(I wrote `7-bit` instead of `7bit` to be consistent with the rest of the man page)